tools/report-converter/codechecker_report_converter/cli.py

```diff
@@ -151,12 +151,19 @@ def __call__(
 
         all_files = []
 
+        had_nonexistent_path = False
```

Comment:
Why do we need this variable? Why don't we `sys.exit(1)` right after `LOG.error()`?

Reply:
I see, thanks!